pkgs/development/python-modules/certifi/default.nix

```diff
@@ -1,23 +1,30 @@
 { lib
 , buildPythonPackage
+, cacert
 , pythonOlder
 , fetchFromGitHub
 , pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "certifi";
-  version = "2022.09.24";
+  version = "2022.12.07";
 
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = "python-certifi";
     rev = version;
-    hash = "sha256-B6LO6AfG9cfpyNI7hj3VjmGTFsrrIkDYO4gPMkZY74w=";
+    hash = "sha256-r6TJ6YGL0cygz+F6g6wiqBfBa/QKhynZ92C6lHTZ2rI=";
   };
 
+  postPatch = ''
+    # Use our systemwide ca-bundle
+    substituteInPlace certifi/core.py \
+      --replace "return _CACERT_PATH" 'return "${cacert}/etc/ssl/certs/ca-bundle.crt"'
```

Comment:
As this is quite critical to still apply in the future I would think we would want to use a patch to know that it is always applied.

Reply:
Good idea.

Reply:
Also codewise maybe replacing https://github.com/certifi/python-certifi/blob/2022.12.07/certifi/core.py#L14-L15 would be cleaner